test-requirements.txt

```diff
@@ -7,3 +7,4 @@ mock==4.0.2; python_version >= '3.6'
 distro; python_version >= '3.8'
 nose
 nose-timer; python_version >= '2.7'
+enum34; python_version < '2.7'
```

Comment:
This is a python >2.6 module backported to 2.6. It contains the `Enum` class, which I'm using for convenience  in `tests.ga.extension_emulator.ExtensionCommandNames`.

Reply:
This is technically a backport for Python versions < 3.4 - https://pypi.org/project/enum34/
Change it to 3.4 instead.

Reply:
I'm surprised the tests passed for 2.7 given you were only installing it on Py2.6! I wonder if the py2.7 travis image already has this module installed in it

Reply:
Hm. That's odd. At least for 2.6, travis doesn't have this module pre installed-- the 2.6 test actually failed before adding this line. (and passed for 2.7+. hence why I only added this dependency for 2.7!)

The docs do very obviously state that enum is "New in version 3.4", so I'll definitely make this change.